src/v0/destinations/braze/util.js

```diff
@@ -374,7 +391,33 @@ const processBatch = (transformedEvents) => {
       headers,
     });
   }
-  const finalResponse = [];
+
+  for (let i = 0; i < subscriptionArrayChunks.length; i += 1) {
```

Comment:
following code blocks are very similar, make it function and reuse for both of them

Reply:
➕